python_modules/libraries/dagster-azure/dagster_azure/adls2/io_manager.py

```diff
@@ -200,9 +200,13 @@ def handle_output(self, context: "OutputContext", obj: Any) -> None:
         self._internal_io_manager.handle_output(context, obj)
 
 
+# ADLS2PickleIOManager used to be named ConfigurablePickledObjectADLS2IOManager, keep this symbol around for backcompat
+ConfigurablePickledObjectADLS2IOManager = ADLS2PickleIOManager
```

Comment:
consider:

```python
@deprecated
class ConfigurablePickledObjectADLS2IOManager(ADLS2PickleIOManager):
   pass
```

So we have the option to delete

Reply:
+1 was going to suggest the same thing

Reply:
ty - that makes a lot more sense